portal-backend/depmap/private/views.py

```diff
@@ -0,0 +1,34 @@
+import logging
+
+from depmap.utilities.sign_bucket_url import sign_url
+from flask import (
+    Blueprint,
+    abort,
+    current_app,
+    redirect,
+)
+from oauth2client.service_account import ServiceAccountCredentials
+
+log = logging.getLogger(__name__)
+
+blueprint = Blueprint(
+    "private", __name__, url_prefix="/private", static_folder="../static"
+)
+
+
+@blueprint.route("/<bucket_name>/<filename>")
+def portal_file_private_url(bucket_name, filename):
+    private_file_buckets = current_app.config.get("PRIVATE_FILE_BUCKETS")
+    # Make sure bucket provided is accepted in valid list of buckets
+    if private_file_buckets is None or bucket_name not in private_file_buckets:
+        abort(404)
+
+    try:
+        credentials = ServiceAccountCredentials.from_json_keyfile_name(
+            current_app.config["DOWNLOADS_KEY"]
```

Comment:
I'm temporarily reusing existing credentials here. They seem to be specific to a project that I'm not planning on using, however, I do not want to use these credentials. I'm a bit confused however, on the process of how to create new credentials tho and would like help on that.

Reply:
I think we talked about this, and I think reusing credentials is fine.

Is this comment left over from before we talked?

Reply:
Yes